web/src/main/java/org/fao/geonet/monitor/link/LinkMonitorInterface.java

```diff
@@ -0,0 +1,44 @@
+package org.fao.geonet.monitor.link;
+
+import jeeves.interfaces.Logger;
+import jeeves.server.resources.ResourceManager;
+import org.fao.geonet.GeonetContext;
+
+public class LinkMonitorInterface {
+    private static ResourceManager resourceManager;
```

Comment:
Should these be static (except perhaps for the logger)?  Gonna get some funny stuff happening if we have more than one `LinkMonitorInterface`.